docs/dev/running/aziot-identityd.md

```diff
@@ -40,45 +47,75 @@ Basic Identity Service configuration file consists of following mandatory settin
 
     The configured value (or the default) will only take effect if the service hasn't been started via systemd socket activation. If it has been started via systemd socket activation, the service will use that socket fd instead.
 
+- `[[principal]]` contains a table of host-process client information. The Identity Service uses the information to provision cloud identities for host processes (using the `name` property), authenticate API clients (using the `uid` property) and manage API access to identity resources (using the `idtype` property).
+
 ## Configuring device provisioning
 
 The `[provisioning.attestation]` section is configured in one of following ways, depending on the provisioning method selected by the device operator:
 
-1. Automated provisioning of IoT Hub device identity using DPS with X.509 attestation - 
+1. No matter which provisioning method is used, you must grant IS access to its master encryption key in KS.
 
-```toml
-	[provisioning]
-	"source" = "dps"
-	"scope_id" = "<ADD DPS SCOPE ID HERE>"
+    `/etc/aziot/keyd/config.d/identityd-principal.toml`
+    ```toml
+    [[principal]]
+    uid = 123 # Replace with output of `id -u aziotid`
+    keys = ["aziot_identityd_master_id"]
+    ```
+
+1. Automated provisioning of IoT Hub device identity using DPS with X.509 attestation -
+
+    `/etc/aziot/identityd/config.toml`
+    ```toml
+    [provisioning]
+    "source" = "dps"
+    "scope_id" = "<ADD DPS SCOPE ID HERE>"
 
     [provisioning.attestation]
-	"method" = "x509"
+    "method" = "x509"
     "registration_id" = "<ADD DPS REGISTRATION ID HERE>" # Optional for X.509 attestation
-	"identity_cert" = "device-id-cert-handle" # Pre-loaded Certificate service handle
-	"identity_pk" = "device-id-key-handle" # Pre-loaded Key service handle
-```
+    "identity_cert" = "device-id-cert-handle" # Pre-loaded Certificate service handle
+    "identity_pk" = "device-id-key-handle" # Pre-loaded Key service handle
+    ```
 
-2. Automated provisioning of IoT Hub device identity using DPS with symmetric key attestation - 
+    You must grant CS access to the `device-id-key-handle` key in KS.
 
-```toml
+    `/etc/aziot/keyd/config.d/certd-principal.toml`
+    ```toml
+    [[principal]]
+    uid = 123 # Replace with output of `id -u aziotcs`
+    keys = ["device-id-key-handle"]
+    ```
+
+2. Automated provisioning of IoT Hub device identity using DPS with symmetric key attestation -
+
+    ```toml
     [provisioning]
-	"source" = "dps"
-	"scope_id" = "<ADD DPS SCOPE ID HERE>"
+    "source" = "dps"
+    "scope_id" = "<ADD DPS SCOPE ID HERE>"
 
     [provisioning.attestation]
     "method" = "symmetric_key"
     "registration_id" = "<ADD DPS REGISTRATION ID HERE>" # Required for symmetric key attestation
     "symmetric_key" = "device-id-symkey-handle" # Pre-loaded Key service handle
```

Comment:
It's not a key handle. It's a key ID. Call it `device-id`

Reply:
Fixed